src/cryptoadvance/specter/templates/includes/sidebar/sidebar.jinja

```diff
@@ -94,6 +94,7 @@
             <a href="https://github.com/cryptoadvance/" target="_blank">
                 <img src="/static/img/ca.png">
             </a>
+	    <span>Specter v<strong>{{ specter.get_version }}</strong></span>
```

Comment:
I'm not really a designer, but I think it would be a bit nicer to just add a small `margin-bottom` or `<br>` at the end of the line so it's not too close to the screen bottom

Reply:
Good call, done. It was `padding-bottom` that worked.